dimager/models.py

```diff
@@ -1,26 +1,37 @@
 from django.db import models
 from django.contrib.auth.models import User
 from django.core.validators import RegexValidator
+from django.utils.encoding import python_2_unicode_compatible
 
 
-class ProfileQuery(models.Model):
+class ActiveImagerManager(models.Manager):
+    '''Return active users, inherit query set'''
     def get_queryset(self):
-        qs = super(ProfileQuery, self).get_queryset()
+        qs = super(ActiveImagerManager, self).get_queryset()
         return qs.filter(user__is_active__exact=True)
 
 
+@python_2_unicode_compatible
 class ImagerProfile(models.Model):
-    user = models.OneToOneField(User)
+    '''Expanding user profile'''
 
-    picture = models.ImageField(upload_to='profile_image', blank=True, max_length=100)
+    user = models.OneToOneField(User, related_name='profile')
+    following = models.ManyToManyField(
+        'self', related_name='_following', symmetrical=False)
```

Comment:
Probably better to give the `related_name` a value that indicates the reverse nature of the relationship.  What's the reverse of `following`?